### DISCUSSION TITLE
How do I change mysite.com to www.mysite.com

### DISCUSSION THREAD

**Discussion by abuzarniazi1** [EasyEngine/easyengine#1770] [General]
I thought It can be done by changing the address to www inside the settings of WP and adding a redirection page rule in CF. But now my site shows 404 Error.
404 Not Found
nginx
Kindly help.
Installed WP using Easyengine on digitalocean ubuntu

**Comment by marcoandrei** [comment]
@abuzarniazi1 

There is no easy way to do that. You need to create a site with www and migrate your WP to there (https://easyengine.io/commands/site/create/).

I recommend you to create the www site with `--type=wp` option and install All-in-One WP Migration & Backup plugin (https://wordpress.org/plugins/all-in-one-wp-migration/) in source and destination sites to migrate from one to the other.

Good luck! Saúde!
